examples/Devices/OLKB/Planck/Planck.ino

```diff
@@ -167,7 +167,7 @@ KALEIDOSCOPE_INIT_PLUGINS(Macros);
 
 void setup() {
   Kaleidoscope.setup();
-  KeyboardHardware.serialPort().begin(9600);
+  Kaleidoscope.serialPort().begin(9600);
```

Comment:
Kaleidoscope.serialPort() -should- just be a thin wrapper around whatever' the device's Serial() is. Different boards expose different devices with different names, but the same API as Serial.

Reply:
Technically, we could move this to the serial driver's `setup()`, if we had such a driver. We currently expose `Serial` or `SerialUSB` directly, without any wrappers outside of `Kaleidoscope.serialPort()` itself.

Another reason I wouldn't move it is because there's a valid case for the end-user using something else than 9600 baud. Especially on faster boards. We wouldn't be able to express that reasonably with a `setup()` method, I think.